### DISCUSSION TITLE
SEO friendly pagination

### DISCUSSION THREAD

**Discussion by pavlentij** [rappasoft/laravel-livewire-tables#498] [Feature Requests]
I've recently found that the pagination block doesn't have links. And when search engines, or monitoring tools (for example, Oh Dear) are crawling/scanning the website, then they cannot get all the links from the table items.

Maybe it's possible by replacing `<button/>` with `<a href=""/>`, so the AJAX requests will still work, but crawlers will also be able to find links to other than first page.

**Comment by rappasoft** [comment]
The pagination is the default Laravel pagination just exported to add dark styles.
